articles/azure-quantum-job-costs.md

```diff
@@ -0,0 +1,59 @@
+
+## Before you start: Understanding job pricing
+Azure Quantum makes solutions available from a number of our partner companies, so the billing details can vary depending on which Provider you would like to use and the pricing plan you select with that provider. Below is some general information on pricing in Azure Quantum, as well as details for each provider.
+
+General pricing guidance
+- You can only use general-purpose Azure Credits with the Microsoft providers. 3rd-party providers (providers that aren't Microsoft) are not eligible. For these providers, you may consider applying to the [Azure Quantum Credits Program](https://aka.ms/aq/credits).
+- Most providers bill based on the resources you consume by running a job, though some also offer subscription plans. For more information, please review the subscription plans available to you in the Azure Portal.
+
+### Honeywell Pricing
+Honeywell uses a credit system called *Honeywell Quantum Credits*, where each job you run consumes a number of credits from your balance based on the number of operations in the job and the number of shots you run. You can find more information on Honeywell Quantum Credits on the [Honeywell Provider page](xref:microsoft.quantum.providers.honeywell).
+
+You may purchase Honeywell Quantum Credits through various plans - please see the Azure Portal for the most up to date list of subscription plans available.
+
+### IonQ Pricing
+IonQ charges based on the number of gates in your program, the complexity of the gates you use, and the number of shots. These units are called `qubit-gate-shots` and you can find more information about how these are calculated on the [IonQ Provider Page](xref:microsoft.quantum.providers.ionq).
+
+IonQ pricing is based on a pay-as-you-go model with no monthly subscription fees. To see the current service costs, please see the Azure Portal.
+
+### 1QBit Pricing
+1QBit charges through either a subscription model or on a pay-as-you-go model based on the runtime of your optimization problem, in seconds. You can find more information about 1QBit's offering on the [1QBit Provider Page](xref:microsoft.quantum.providers.optimization.1qbit).
+
+### Microsoft Optimization Pricing
+Microsoft Optimization Solutions charges based on the runtime of your optimization problem, in seconds. Per-second rates are based on the plan you select and the volume of optimization jobs you run within a month. Please see the [Microsoft Optimization Pricing Page](https://azure.microsoft.com/pricing/details/azure-quantum/) for more information.
+
+## After you run: Job Cost Reporting
+After you run a job, Azure Quantum makes available detailed cost estimates for supported providers. You can use this information to understand the cost of individual jobs. The information is presented as an estimate only, and you should refer to your final bill for the exact charges including relevant taxes.
```

Comment:
I'm worried customers might think that we're only estimating how much the job consumed. Instead of "The information is presented as an estimate only", can we say something along the lines of _Note that the final charges displayed no your bill might be slightly different due to factors like added taxes._

Reply:
I like it! thanks

Reply:
hmm I'm not seeing the update in the last iteration.